Apps/AdminWebClient/src/ClientApp/src/components/core/CommunicationTable.vue

```diff
@@ -228,11 +231,10 @@ export default class CommunicationTable extends Vue {
         }
     }
 
-    private editBanner(item: Communication) {
+    private edit(item: Communication) {
         this.editedIndex = this.communicationList.indexOf(item);
         this.editedBanner = item;
```

Comment:
Should this switch based on the communication type? That will make sure the banner object does not load the email one

Reply:
not necessarily... since both the banner and email are just variations of the communication model with different relevant fields we can simplify it by setting both the banner and the email to the selected item. they get cleared upon closing the modal anyway.